debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -267,6 +302,10 @@ public void dispatchTransactionMessage(SourceRecord record) throws InterruptedEx
         queue.enqueue(new DataChangeEvent(record));
     }
 
+    public void dispatchSchemaChangeMessage(SourceRecord record) throws InterruptedException {
```

Comment:
How is it different from the method above?

Reply:
It is the same but the trigger to generate the message is different. We could merge it into a single method but I like keeping it separated - it is easy to add further logging/metrics etc.

Reply:
Ok, so this should be private and also named `enqueueSchemaChangeMessage()`. Similar for `dispatchTransactionMessage()` right above. Those methods act on a different level than the dispatchXyz ones.